docs/mkdocs/docs/runtime_config.md

```diff
@@ -82,21 +82,49 @@ If only `NumCPUThreads` is set, `NumIOThreads` will still default to x1.5 `NumCP
  
 * `version` - contains information about versions being read, created, or destroyed, and traversal of the [version layer](/technical/on_disk_storage#version-layer) linked list
 * `storage` - contains information about individual operations that interact with the storage device (read object, write object, delete object, etc)
 
-The available log levels in decreasing order of verbosity are are `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, `CRITICAL`, `OFF`. By default, all streams are set to the `INFO` level.
+The available log levels in decreasing order of verbosity are are `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, `CRITICAL`, `OFF`. 
+By default, all streams are set to the `INFO` level.
 
-There are two ways to configure log levels. The first is via environment variables e.g. `ARCTICDB_version_loglevel=DEBUG`. All of the streams can be configured together via `ARCTICDB_all_loglevel=DEBUG`. The second is in code by calling `set_log_level` from the `arcticdb.config` module. This takes two optional arguments:
+There are two ways to configure log levels: 
+
+1. **Setting an environment variable**: `ARCTICDB_<stream>_loglevel=<level>`, for example: `ARCTICDB_version_loglevel=DEBUG`. All streams can be configured together via `ARCTICDB_all_loglevel`. 
+2. **In code**: Calling `set_log_level` from the `arcticdb.config` module. This takes two optional arguments:
 
 * `default_level` - the default level for all streams. Should be a string such as `"DEBUG"`
 * `specific_log_levels` - a dictionary from stream names to log levels used to override the default such as `{"version": "DEBUG""}`.
 
 If both environment variables are set, and `set_log_level` is called, then the latter takes priority.
 
-Note that all logging from ArcticDB goes to `stderr`, and this is not configurable.
+S3 logging can also be enabled by setting the environment variable `ARCTICDB_AWS_LogLevel_int=6`, which will output all S3 logs to a file in the present working directory. 
+See the [AWS documentation](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/logging.html) for more details.
+
+### Logging destinations
+
+By default, all logging from ArcticDB goes to `stderr`. This can be configured using the `set_log_level` method.
+
+To configure logging to only a file:
+
+```
+from arcticdb.config import set_log_level
+set_log_level(console_output=False, file_output_path="/tmp/arcticdb.log")
+```
 
-S3 logging can also be enabled by setting the environment variable `ARCTICDB_AWS_LogLevel_int=6`, which will output all S3 logs to a file in the present working directory. See the [AWS documentation](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/logging.html) for more details.
+To configure logging to both `stderr` and a file:
 
+```
+from arcticdb.config import set_log_level
+set_log_level(console_output=True, file_output_path="/tmp/arcticdb.log")
```

Comment:
The code calls the parameter `file_output`

Reply:
Good spot. Fixed locally, tested, but didn't push. Pushed now.